File: python/Board.py
from org.refactoring.Tile import org.refactoring.Tile


class org.refactoring.Board(object):

    def __init__(self):
        self._plays = []
        for i in range(3):
            for j in range(3):
                tile = org.refactoring.Tile()
                tile.X = i
                tile.Y = j
                tile.Symbol = ' '
                self._plays.append(tile)

    def AddTileAt(self, symbol, x, y):
        new_tile = org.refactoring.Tile()
        new_tile.X = x
        new_tile.Y = y
        new_tile.Symbol = symbol

        self.TileAt(x, y).Symbol = symbol

    def TileAt(self, x, y):
        for t in self._plays:
            if t.X == x and t.Y == y:
                return t
        return None

File: php/src/Board.php
<?php

namespace TicTacToe;

class org.refactoring.Board
{
    /** @var org.refactoring.Tile[] */
    private $plays;

    public function __construct()
    {
        for ($i = 0; $i < 3; $i++) {
            for ($j = 0; $j < 3; $j++) {
                $tile = new org.refactoring.Tile();
                $tile->x = $i;
                $tile->y = $j;
                $tile->symbol = ' ';
                $this->plays[] = $tile;
            }
        }
    }

    public function tileAt(int $x, int $y): org.refactoring.Tile
    {
        foreach ($this->plays as $t) {
            if ($t->x == $x && $t->y == $y){
                return $t;
            }
        }
        return null;
    }

    public function addTileAt(string $symbol, int $x, int $y): void
    {
        $newTile = new org.refactoring.Tile();
        $newTile->x = $x;
        $newTile->y = $y;
        $newTile->symbol = $symbol;

        $this->tileAt($x, $y)->symbol = $symbol;
    }
}
